### PR TITLE
regressions done

### PR DESCRIPTION
**Pull request by Patricia888**

### CODE REVIEW

**Comment by Patricia888** [comment]
- [ x] Show the first 5 rows of the dataset
- [x ] Show the description and the info on the dataset, using appropriate Pandas functions
- [ x] Use seaborn jointplot function to see which fields correlate well with the "Yearly Amount Spent" column. Write your findings.
- [ x] Make a summary plot of feature relationships using pairplot function of Seaborn
- [ x] Create two dataframes: one for the target variable ("Yearly Amount Spent"), the other - containing all the rest of numerical features
- [ x] Split the data into a training and test sets. Make a test set size 0.3 and random seed 123 (so the results are consistent for all students)
- [ x] Fit a regression model on the training set
- [x ] Print out the coefficients of the model
- [ x] Make a prediction of the target variable from features dataframe
- [ x] Calculate the Mean Average Error (using sklearn.metrics module)
- [ x] Using Seaborn distplot show the histogram of the residuals - differences between the target variable and predicted target variable
- [ x] Answer the main question: How should we allocate the engineering budget between website development and app development?
